### DISCUSSION TITLE
[Video] Are quasars really going extinct?

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#260] [Astrophysics and Physics]
Here's my latest video:

**"Are quasars really going extinct?"**

https://www.youtube.com/watch?v=W_9QEsMGz7c

Most of the information is covered in the #210 thread. It shows the data in various studies. SDSS, DESI.

The gist is that in a nonexpanding universe, they wouldn't be distributed the way they are. I look at some of the different ways the brightness of quasars could be overestimated.

**Comment by RedshiftDrift** [comment]
(Note: I use "_BB" to label a word that is to be interpreted in a Big Bang Cosmology paradigm, and "_TL" to label a word that is to be interpreted in a Tired-Light Cosmology paradigm.)

'Quasars_BB' are not going extinct.  Their distribution_BB (as plotted by @mikehelland) calculated_BB from redshift measurements is not representative of the real distribution_TL of these objects in space.

'Quasars_BB' are ordinary_TL galaxies that are nearly aligned behind other galaxies.  When the light emitted by a background galaxy passes through a moving plasma filament that is part of a foreground galaxy, the tired-light (T-L) redshift is enhanced due to the high density of the plasma.  This gives a higher_TL redshift to the background galaxy, and since the number of photons is conserved in T-L redshift, the background galaxy is calculated_BB to be abnormally luminous_BB, which is why it is identified as a 'quasar_BB'.  (See definition on Wikipedia: "A quasar_BB is an extremely luminous_BB active_BB galactic nucleus.")  The small number (millions) of 'quasars_BB' (compared to billions of galaxies) is only due to the low probablilty of such alignments.[^1]

The low number of 'quasars_BB' at small distances is only because a galaxy (identified as a 'quasar_BB') has less chance of being behind an even closer galaxy.

This makes 'quasars_BB' appear as if they have a *large intrinsic_BB redshift* (http://iopscience.iop.org/0004-637X/566/2/705), but these_TL objects are *"a lot closer than their redshifts imply_BB"*. See Bell [http://arxiv.org/abs/astro-ph/0602242](http://arxiv.org/abs/astro-ph/0602242).

[^1]: Halton Arp initially interpreted 'quasar_BB'-galaxy associations as a physical association, but it is in reality an *optical_TL association* on the sky.  The galaxy_TL labeled as a 'quasar_BB' is at a greater distance than its parent galaxy, but not at the distance_BB indicated_BB by its redshift.  The more recent work of Arp, Fulton and Hartnett demonstrates this angular association in the sky.
Fulton, C.C., Arp, H.C., and Hartnett, J.G., "Physical association and periodicity in quasar_BB families with SDSS and 2MRS," Astrophysics and Space Science 363:134, 2018.
Fulton, C.C. and Arp, H.C., The 2dF Redshift Survey. I. Physical association and periodicity in quasar_BB families, Astrophys. J. 754:134, 2012; iopscience.iop.org/article/10.1088/0004-637X/754/2/134/meta

**Reply by HanDeBruijn** [reply]
Isn't your writing contradictory to [Wikipedia](https://en.wikipedia.org/wiki/Quasar), where it is stated that: _A **quasar** is an extremely [luminous](https://en.wikipedia.org/wiki/Luminosity) [active galactic nucleus](https://en.wikipedia.org/wiki/Active_galactic_nucleus) (AGN)_ ?

**Reply by RedshiftDrift** [reply]
@HanDeBruijn my apologies for the confusing note.  I am writing from the point of view of a *"Tired-Light cosmology"*, a paradigm that is incommensurate with the *"Big Bang cosmology"* paradigm.  Since the same words don't have the same meaning in each paradigms, this gives the impression of a contradiction.

I've [updated my message](https://github.com/orgs/a-cosmology-group/discussions/260#discussioncomment-10479543) by specifying which paradigm is implied when the word is ambiguous.  Hopefully this clarifies things.

**Reply by HanDeBruijn** [reply]
Okay, but what then is an **AGN**_TL, if it is not a quasar_TL ?

**Reply by RedshiftDrift** [reply]
Second line: "'Quasars_BB' are ordinary_TL galaxies..."

**Reply by sahil5d** [reply]
What is the evidence against a quasar being an active galactic nucleus?

**Comment by mikehelland** [comment]
@HanDeBruijn 

The difference between an AGN and quasar is brightness. According to the Quasar and AGN Vernon 12th edition it's brighter than -23 in absolute magnitude.

https://cdsarc.cds.unistra.fr/ftp/cats/VII/248/ReadMe

    qso.dat       142     85221   Quasars (brighter than absolute magnitude -23)
    agn.dat       142     21737   Active galaxies (fainter than -23)

In the 13th edition, it's -22.25.

https://cdsarc.cds.unistra.fr/ftp/VII/258/ReadMe

    qso.dat       142    133335  *Quasars (brighter than absolute magnitude -22.25)
    agn.dat       142     34231  *Active galaxies (fainter than -22.25)

So it's basically just a brightness thing. 

If you look at this one:

https://en.wikipedia.org/wiki/3C_273

It's at a magnitude of -26.7, so super bright. This one is so close, we can see the galaxy around it, so we call it an AGN.

If it's farther than that, we call it a quasar. That seems to be the deal.

Also, if you watch the video, you'll see some quasars in our best telescopes are actually more than one object: a double quasar, or an intermingling star or galaxy.

**Reply by HanDeBruijn** [reply]
From the above by @RedshiftDrift:
>'Quasars_BB' are ordinary_TL galaxies that are nearly aligned behind other galaxies.

From [Seeing Red](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/968368905_Arp.pdf) by Halton Arp (p.55,56):
>Another example of the penalty people pay for not using operational definitions is the term "AGN" (active galaxy nuclei). [ .. ]
Eventually, everyone came to believe that quasars had host galaxies. [ .. ]
When the Space Telescope started taking high-resolution pictures of quasars, John Bahcall called a press conference to report that a number of them did not have any host galaxies at all! Gasp! Naked quasars!! [ .. ]
It would now be natural to describe an empirical sequence of quasar development, from initially point-like objects at relatively faint apparent magnitudes, gradually transforming into lower-redshift compact objects with "fuzz" around their perimeters, then into small, high surface-brightness galaxies with more material around them and, finally, normal, quiescent galaxies. [ .. ]
The juggernaut has continued to gather momentum to the present day.

**Reply by budrap00** [reply]
Taken at face value, Arp's quasar to galaxy evolution provides something of great significance to non-BB cosmology - an observable account of galaxy formation. The BB, of course, provides an account of formation via the accretion method but there is no direct empirical evidence supporting that model. In this, the accretion model is like all the other fundamental elements of LCDM which are either events buried in a mythological past or simply _ad hoc_, empirically undetected fitting parameters.

There is a reason that telescope time for direct quasar-galaxy study has been denied since Arp's time. Arp's model of galaxy formation is based on empirical observations and presents a direct refutation of the BB model. Tossing aside Arp's model based on the same spurious reasoning employed by BB proponents in denying telescope time to investigate the quasar-galaxy relationship simply denies non-BB proponents a potentially conclusive refutation of the standard model.

Aside from fatally undermining the BB model, galaxy to quasar evolution is the only galaxy evolution model besides BB-accretion that I am aware of. So, if you're going to dismiss both Arp and the BB theory you are, as far as I know, left with _no_ account of galaxy formation. That would seem to be a serious handicap for any proposed alternative to the standard model.

**Comment by ExpEarth** [comment]
If my shell model is correct, then the apparent abundance of quasars at higher redshifts could be just due to the greater density of gas in the shell region.  My recent shell model version features the relativistic Ni solutions, which would correlate to the universe having a lower density in the centre and a higher one at the shell.  Higher gas density at high redshifts translates to bigger galaxies, with bigger supermassive black holes and thus brighter quasars.  The Hubble redshift would not be due to expansion, but to a gravitational redshift induced by the shell.

There are other points where the mainstream explanation of quasars is likely incorrect (the one in Wikipedia article "Quasars").  I don't think the mainstream explanation for the quasar luminosity, energy released by matter falling onto the accretion disk, is adequate.  That process would be too irregular.  Here my explanation, the Hubble luminosity, would be the better one.

There is still the possibility, however, that there is an intrinsic redshift component in quasars, perhaps due to a mechanism similar to the one by @RedshiftDrift .  If there were such a mechanism, this would also add to an apparent shortage of local quasars and a superabundance of remote ones.  The intrinsic and Hubble redshifts would have to be multiplied together, not summed, and this magnifies the anomaly.

If there were an intrinsic quasar redshift, then maybe it could account for Arp's observations that the 'ejected quasar' is redshifted.  If the intrinsic redshift occurs in the quasar jet, might it not persist in the jet's product, a new small galaxy?  So it might be possible to simplify the model by @RedshiftDrift , in that it would not be a case of galaxy alignments, but similar processes occurring in the jet and the nascent galaxy.  Just some thoughts.

General conclusion is quasars are not going extinct!

**Reply by HanDeBruijn** [reply]
>The intrinsic and Hubble redshifts would have to be multiplied together, not summed, and this magnifies the anomaly.

That's essentially correct. Let subscripts $i$ and $t$ denote _intrinsic_ and _tired-light_ respectively, then

$$
1+z = (1+z_i)(1+z_t) = e^{H_i D/c_0}\,e^{H_t D/c_0} = e^{(H_i+H_t) D/c_0} = e^{H_0 D/c_0}
$$

with $z=$ redshift, $H=$ Hubble parameter, $c_0=$ standard lightspeed, $D=$ distance.
It follows that the total Hubble is the sum of the intrinsic and the tired-light one:

$$
H_0 = H_i + H_t
$$

**Comment by RedshiftDrift** [comment]
"Lonely quasars in the early universe cannot be explained"
https://www.earth.com/news/lonely-quasars-in-the-early-universe-cannot-be-explained/

That's not quite accurate... They *can* be explained, but not with LCDM!

The published paper
"EIGER. VI. The Correlation Function, Host Halo Mass, and Duty Cycle of Luminous Quasars at z ≳ 6"
https://iopscience.iop.org/article/10.3847/1538-4357/ad778b
 The Astrophysical Journal, Volume 974, Number 2

> Such short quasar activity timescales challenge our understanding of early supermassive black hole growth and provide evidence for highly dust-obscured growth phases or episodic, radiatively inefficient accretion rates.
